kotlinpoet/gradle.properties

```diff
@@ -2,3 +2,4 @@ POM_ARTIFACT_ID=kotlinpoet
 POM_NAME=KotlinPoet
 POM_DESCRIPTION=Use beautiful Kotlin code to generate beautiful Kotlin code.
 POM_PACKAGING=jar
+kotlin.mpp.applyDefaultHierarchyTemplate=false
```

Comment:
[Here](https://github.com/square/kotlinpoet/pull/1959#discussion_r1732580303)'s why: because some of the source code sets are customised in `build.gradle.kts` and their structure is not the default one.

Reply:
Thanks!

Reply:
We still will want the default hierarchy which can be customized. For example: https://github.com/JakeWharton/mosaic/blob/167f183142c7ce83bc20d0ea104021c5b3149830/mosaic-runtime/build.gradle#L8-L17. We can add it in a follow-up.